src/NQueenTest.java

```diff
@@ -0,0 +1,37 @@
+import static org.junit.Assert.*;
+
+import org.junit.*;
+public class NQueenTest
+{
+	NQueen obj;
+	@Before
+	public void initializeClass() throws Exception
+	{
+		obj=new NQueen();
+	}
+	@Test
+	public void NQueenProblemTest()
+	{
+		try
+		{
+			obj.createMatrix(3);
+		}
+		catch(Exception e)
+		{
+			assertEquals("No Solution exist for N smaller than 3",e.getMessage());
+		}
+	}
+	
+	@Test
+	public void NQueenProblemTest_negative_value()
```

Comment:
This case is the similar to above the one. So you could remove it.

Reply:
done